src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj

```diff
@@ -14,7 +14,7 @@
   <Target Name="AddFrameworkFilesToPackage" DependsOnTargets="ResolveLibrariesFromLocalBuild" BeforeTargets="GetFilesToPackage">
     <ItemGroup>
       <ReferencePath Include="@(LibrariesRefAssemblies)" />
-      <DocFilesToPackage Include="$(ArtifactsBinDir)/docs/%(LibrariesRefAssemblies.FileName).xml" Condition="Exists('$(ArtifactsBinDir)/docs/%(LibrariesRefAssemblies.FileName).xml')"/>
+      <DocFilesToPackage Include="$(XmlDocDir)/%(LibrariesRefAssemblies.FileName).xml" Condition="Exists('$(XmlDocDir)/%(LibrariesRefAssemblies.FileName).xml')"/>
```

Comment:
nit: you don't the slash here as `XmlDocDir` already has a trailing slash, but we can keep it as msbuild will just ignore it.